defaults/perftune_results.json

```diff
@@ -0,0 +1,179 @@
+{
+  "aws":
+  {
+    "x86_64": {
+      "2": {
+        "get-cpu-mask": "0x00000003",
+        "get-irq-cpu-mask": "0x00000003",
+        "dump-options-file": {"cpu_mask": "0x00000003", "irq_cpu_mask": "0x00000003"}
+      },
+      "4": {
+        "get-cpu-mask": "0x0000000f",
+        "get-irq-cpu-mask": "0x0000000f",
+        "dump-options-file": {"cpu_mask": "0x0000000f", "irq_cpu_mask": "0x0000000f"}
+      },
+      "8": {
+        "get-cpu-mask": "0x000000fe",
+        "get-irq-cpu-mask": "0x00000001",
+        "dump-options-file": {"cpu_mask": "0x000000ff", "irq_cpu_mask": "0x00000001"}
+      },
+      "12": {
+        "get-cpu-mask": "0x00000fbe",
+        "get-irq-cpu-mask": "0x00000041",
+        "dump-options-file": {"cpu_mask": "0x00000fff", "irq_cpu_mask": "0x00000041"}
+      },
+      "16": {
+        "get-cpu-mask": "0x0000fefe",
+        "get-irq-cpu-mask": "0x00000101",
+        "dump-options-file": {"cpu_mask": "0x0000ffff", "irq_cpu_mask": "0x00000101"}
+      },
+      "24": {
+        "get-cpu-mask": "0x00ffeffe",
+        "get-irq-cpu-mask": "0x00001001",
+        "dump-options-file": {"cpu_mask": "0x00ffffff", "irq_cpu_mask": "0x00001001"}
+      },
+      "32": {
+        "get-cpu-mask": "0xfffefffe",
+        "get-irq-cpu-mask": "0x00010001",
+        "dump-options-file": {"cpu_mask": "0xffffffff", "irq_cpu_mask": "0x00010001"}
+      },
+      "48": {
+        "get-cpu-mask": "0x0000ffff,0xfcfffffc",
+        "get-irq-cpu-mask": "0x03000003",
+        "dump-options-file": {"cpu_mask": "0x0000ffff,0xffffffff", "irq_cpu_mask": "0x03000003"}
```

Comment:
won't these results depend on instances generation? e.g. when AWS introduces i5i, we start to fail these tests .
also, how do you know these are correct :sweat_smile: ?

Reply:
I don't know if it's correct or not

this is a PR started by shlomo, and I've trying to see what we can get out of it.

I'm playing with throwing this file away, and just validate there are enough bits on, on each mask.

Reply:
I don't remember what was the outcome of our call with Shlomo, but I recall I wasn't happy with the methodology here.
Need to refresh my memory with the original PR.

Reply:
This file is going away, and now I'll do the calculation of what is expected inside the code

There won't be exact masks, but the number of CPUs for scylla and number of CPUs for IRQ